site-content/source/modules/ROOT/pages/development/how_to_commit.adoc

```diff
@@ -162,14 +176,11 @@ while you are resolving the issue.
 ====
 The fastest way to get a patch from someone’s commit in a branch on GH -
 if you don’t have their repo in remotes - is to append .patch to the
-commit url, e.g. curl -O
-https://github.com/apache/cassandra/commit/7374e9b5ab08c1f1e612bf72293ea14c959b0c3c.patch
+commit url, e.g. `+curl -O https://github.com/apache/cassandra/commit/7374e9b5ab08c1f1e612bf72293ea14c959b0c3c.patch+`
 ====
 
 [TIP]
 .Tip
 ====
-`+git cherry-pick -n <sha-of-X.X-commit>+` can be used in place of the
-`+git format-patch -1 <sha-of-X.X-commit> ; git apply -3 <sha-of-X.X-commit>.patch+`
-steps.
+`+git format-patch -1 <sha-of-X.X-commit> ; git apply -3 <sha-of-X.X-commit>.patch+` can be used in place of the `+git cherry-pick -n <sha-of-X.X-commit>+` steps.
```

Comment:
An observation: for me this advice did not work as is - the patch file name is not a SHA
```
git --version
git version 2.48.1

git format-patch -1 c295c33a26
0001-Optimize-UTF8Validator.validate-for-ASCII-prefixed-S.patch
```